roles/prune_catalog/README.md

```diff
@@ -44,6 +43,109 @@ The following application must be already present on the system.
     pc_opm_args: "--skip-tls-verify=true"
 ```
 
+All channels for the listed operators are mirrored
+```ShellSession
+$ DOCKER_CONFIG=/home/<user>/.docker oc-mirror list operators --catalog <my-local-registry>:4443/pruned-catalog:latest --package compliance-operator
+NAME                 DISPLAY NAME         DEFAULT CHANNEL
+compliance-operator  Compliance Operator  stable
+
+PACKAGE              CHANNEL      HEAD
+compliance-operator  4.7          compliance-operator.v0.1.32
+compliance-operator  release-0.1  compliance-operator.v0.1.61
+compliance-operator  stable       compliance-operator.v1.3.1
+```
+
+Prune a dictionary for specific channel
+```yaml
+- name: "Prune a catalog specific with channels"
+  include_role:
+    name: redhatci.ocp.prune_catalog
+  vars:
+    pc_source_catalog: "registry.redhat.io/redhat/redhat-operator-index:v4.13"
+    pc_destination_catalog: "<my-local-registry>:4443/pruned-catalog:latest"
+    pc_operators:
+      compliance-operator:
+        channel: stable
+      file-integrity-operator:
+        channel: stable
+      cluster-logging:
+        channel: stable
+    pc_opm_args: "--skip-tls-verify=true"
+```
+
+Only the specific channel for the listed operators are mirrored
+```ShellSession
+$ DOCKER_CONFIG=/home/<user>/.docker oc-mirror list operators --catalog <my-local-registry>:4443/pruned-catalog:latest 
+NAME                     DISPLAY NAME               DEFAULT CHANNEL
+cluster-logging          Red Hat OpenShift Logging  stable
+compliance-operator      Compliance Operator        stable
+file-integrity-operator  File Integrity Operator    stable
+
+$ DOCKER_CONFIG=/home/<user>/.docker oc-mirror list operators --catalog <my-local-registry>:4443/pruned-catalog:latest --package compliance-operator
+NAME                 DISPLAY NAME         DEFAULT CHANNEL
+compliance-operator  Compliance Operator  stable
+
+PACKAGE              CHANNEL  HEAD
+compliance-operator  stable   compliance-operator.v1.3.1
+```
+
+Prune a dictionary for specific channel
+```yaml
+- name: "Prune a all and/or single channels"
```

Comment:
Just wonder if this is a typo `Prune a all ...` maybe it's just `Prune all ...`

Reply:
yes, it was a typo. Thank!